src/pages/PlottingProgress.vue

```diff
@@ -229,23 +223,23 @@ export default defineComponent({
         util.errorLogger("PLOTTING PROGRESS | Farmer start error!")
       }
       util.infoLogger("PLOTTING PROGRESS | farmer started")
-      const networkSegmentCount = config.segmentCache.networkSegmentCount
-      this.networkSegmentCount = networkSegmentCount
+
       this.plottingData.allocatedGB = config.plot.sizeGB
-      this.localSegmentCount = await this.$client.getLocalSegmentCount()
-      do {
-        await new Promise((resolve) => setTimeout(resolve, 2000))
-        this.localSegmentCount = await this.$client.getLocalSegmentCount()
-      } while (this.localSegmentCount < this.networkSegmentCount)
 
+      this.syncState = (await this.$client.getSyncState()).toJSON() as unknown as SyncState;
```

Comment:
If we are going to refactor it, i think its fine. 
As Nazar suggested, maybe we can have `TODOs` in such places to remind ourselves and the reviewer that it will be refactored. We can use that approach in the next PR's

Reply:
I want to clarify how far we're ready to go with the refactoring of this version (before we switch to React) at the meeting tomorrow. Otherwise, I would have to add too many TODOs